.github/workflows/windows.yml

```diff
@@ -15,20 +15,6 @@ jobs:
     - name: test
       run: cd build ; ctest -j 10 -C Debug --exclude-regex "test-unicode" --output-on-failure
 
-  clang9:
```

Comment:
Why did you remove Clang 9?

Reply:
The commit message explains why, see e2264601542bd29a88fc85570b95c4e897eb2781.

Reply:
Could the problem be that windows-latest doesn't remain the same? Newer versions of Visual studio's standard c++ library also require newer versions of clang, so when testing llvm on windows, a fixed version of clang should probably be paired with a fixed version of the VS toolchain.